node/narwhal/src/helpers/channels.rs

```diff
@@ -58,7 +58,7 @@ pub fn init_consensus_channels<N: Network>() -> (ConsensusSender<N>, ConsensusRe
     (sender, receiver)
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
```

Comment:
This is actually not needed for the code to function after all but there's no reason not to impl it afaict.